docs/reference/run.md

```diff
@@ -21,7 +21,7 @@ executes `docker run`, she starts a process with its own file system,
 its own networking, and its own isolated process tree.  The
 [*Image*](/terms/image/#image) which starts the process may define
 defaults related to the binary to run, the networking to expose, and
-more, but `docker run` gives final control to the operator who starts
+more, but `docker run` gives additional control to the operator who starts
```

Comment:
@SpencerBrown @cpuguy83 try this new intro --- WDYT?

--> https://gist.github.com/moxiegirl/3acf6eb35fbe777d1ffd

# Docker run reference

Docker runs processes in isolated containers. A container is a process which
runs on a host. The host may be local or remote. When an operator executes
`docker run`, the container process that runs is isolated in that it has its
own file system, its own networking, and its own isolated process tree separate
from the host. 

This page details how to use the `docker run` command to define the container's
 resources at runtime. 

## General form

The basic `docker run` command takes this form:

    $ docker run [OPTIONS] IMAGE[:TAG|@DIGEST] [COMMAND] [ARG...]
    
The `docker run` command must specify an [*IMAGE*](/terms/image/#image) to derive
the container from. An image developer can define image defaults related to

 * detached or foreground running
 * container identification
 * network settings
 * runtime constraints on CPU and memory
 * privileges and LXC configuration

With the `docker run [OPTIONS]` an operator can add to or override the image
defaults set by a developer. And, additionally, operators can override nearly
all the defaults set by the Docker runtime itself. The operator's ability to
override image and Docker runtime defaults is why
[*run*](/reference/commandline/cli/#run) has more options than any other
`docker` command.

>**Note**: Depending on your Docker system configuration, you may be required to
preface the `docker run` command with `sudo`. To avoid having to use `sudo` with
the `docker` command, your system administrator can create a Unix group called
`docker` and add users to it. For more information about this configuration,
refer to the Docker installation documentation for your operating system.